components/README.md

```diff
@@ -1 +1,38 @@
-# Here put a description about your project
\ No newline at end of file
+# Welcome to SUI
+
+SUI is an open-source, high-quality library of reusable React components that empower teams to craft any product with ease.
+
+## Declarative approach:
+
+SUI is Declarative, Intuitive, Consistent and Product-agnostic. Every component should allow its customisation through React Props.
+
+If you need to add or adjust any of the following for a component, you can send a PR directly without previous notice.
+
+- Font sizes, colour and weight
+- Border visibility, size and colour
+- Border radius
+- Box-Shadow style
+- Background colour
+- Paddings and Margins
+- Icon and icon colour
+
+## Contribute
+
+SUI is an open source project, feel free to send a PR, or use one of the templates below
+
+- Report a bug / issue:
+- Propose a new component:
+- Improve and existing component:
+
+## Adevinta's Design System · Resources
+
+SUI is a fundamental part of Adevinta's Design System, wich also includes UI Kits, Tools and Processes to facilitate how we collaborate and share knowledge across teams and projects, bringing support for Designers, Product Managers and Developers.
+
+- Download the Sketch UI-Kit:
+- SUI tools: 
+- Get in touch: Design-Ops, Development:
+- Status Dashboard: https://pages.github.mpi-internal.com/scmspain/sui-dashboard/
```

Comment:
I think this URL might not be accesible from external office IPs

Reply:
yeap... I'm aware, I still think it's a good link to have there, even if it's a dead-end for external users. What do you think?

Reply:
Maybe you could add a warning or something

Reply:
Sure, that's a good idea
Changes done, please now